sql/2019/08_Security/08_43.sql

```diff
@@ -0,0 +1,11 @@
+# 08_43: HTTP vs HTTPS
+SELECT
+  client,
+  SUM(IF(url LIKE 'https://%', 1, 0))  AS https,
```

Comment:
FYI, using semantic functions
```suggestion
  SUM(COUNTIF(STARTS_WITH(url, 'https://')))  AS https,
```

Reply:
updated this as well